#! /usr/bin/env python
# Gamepad configuration

PACKAGE='hanse_depthengine'
import roslib; roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()
#       Name                                   Type      Level Description                             Default  Min   Max
gen.add("publish_frequency",                   double_t, 0,    "how often to send data to server",     10,      0,    1000)
gen.add("min_depth_pressure",                  int_t,    0,    "minimal pressure value",               950,     900,  1100)
gen.add("max_depth_pressure",                  int_t,    0,    "maximal pressure value",               1500,    1300, 1700)
gen.add("depth_pid_enabled_at_start",          bool_t,   0,    "depth pid enabled at start",           True)

exit(gen.generate(PACKAGE, "depthengine_node", "Depthengine"))
